huaweicloud/services/eip/resource_huaweicloud_vpc_eip.go

```diff
@@ -19,36 +25,51 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
-func ResourceVpcEIPV1() *schema.Resource {
```

Comment:
it's not recommended to change the name as the function has been exported.

Reply:
done. the change has been reverted.